spec/newgistics/order_spec.rb

```diff
@@ -22,7 +23,7 @@
       end
     end
 
-    vcr_options = { cassette_name: 'order/save/failure' }
+    vcr_options = { cassette_name: 'order/save/failure', record: :new_episodes }
```

Comment:
should we remove this `record: :new_episodes`?

Reply:
Yes, definitely, thanks for catching this!